hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2684,6 +2699,18 @@
               "chip_sw_uart_smoketest",
             ]
     }
+    {
+      name: chip_sw_signed
+      desc: '''Run some chip-level tests with mask ROM.
+
+            In addition to mask ROM E2E tests, we select some of tests defined
+            in this file to sign, and run via mask ROM instead of test ROM. We
+            need to ensure our test infrastructure and mask ROM can boot and run
+            some of the same tests our test ROM can.
+            '''
+      milestone: V2
+      tests: ["chip_sw_mask_rom_uart_smoketest_signed"]
+    }
```

Comment:
Same comment as above.

Reply:
Actually nevermind - if this just says "we want to run at least one (or few) test with a signed image, doesn't matter which, then this LGTM.

Reply:
SG, I updated the description to reflect "one (or a few)" tests only.